data/model/oci/tag.py

```diff
@@ -184,8 +184,8 @@ def list_repository_tag_history(
         .offset(page_size * (page - 1))
     )
 
-    if specific_tag_name is not None:
-        query = query.where(Tag.name == specific_tag_name)
+    if tag_names is not None:
+        query = query.where(Tag.name << tag_names)
```

Comment:
Will this make use of the indices. We're changing the query from `... WHERE X = Y` to `... WHERE X in SOMELIST`.

Reply:
When this query is executed, is it of the same type as before? Can you add a small test for this.

Reply:
Talked to @kwestpharedhat about the where query. There's a possibility of performance impact. For mirroring the `tag_names` list can be in the thousands so i'm rethinking adding this. We may want to go with the safer option of not modifying this query and instead filtering on the returned list.

Reply:
updated